### ISSUE TITLE
Einbettungscode Dialog a11y issues

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#625]
- dimension buttons cannot be accessed with the keyboard
- screen reader reads out the table underneath the modal
- Focusing after pressing the copy button causes the screen reader to read out the embed code and does not give the feedback that the code has been copied to the clipboard -> feedback should be in a live area
- is there a good reason why the embed code is in a textarea?

**Comment by dennis531** [comment]
> * Focusing after pressing the copy button causes the screen reader to read out the embed code and does not give the feedback that the code has been copied to the clipboard -> feedback should be in a live area

Still open, as I didn't know how to solve this problem without hacks.

**Comment by Arnei** [comment]
All issues should be fixed now
